react-sharing-state/my-app/src/Container.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { useState } from 'react';
+
+/**
+ * A container of items.
+ * One item is displayed at a time, with buttons to flip through them:
+ * - Next and Prev scroll forward and backward one item
+ * - An array of buttons scroll to a specific item
+ * TODO: The buttons don't work!
+ */
+export default function Container({ items }) {
+  const [isActive, setIsActive] = useState(0);
```

Comment:
Usually you would use the prefix 'is' only for booleans. `activeIndex` might be a better name.